Library/ViewModels/ProjectPamphletContentViewModelTests.swift

```diff
@@ -398,14 +398,19 @@ final class ProjectPamphletContentViewModelTests: TestCase {
     self.loadMinimalProjectIntoDataSource.assertValues([project], "Nothing new emits.")
   }
 
-  func testLoadProjectIntoDataSource_CreatorDetailsLoaded() {
+  func testLoadProjectIntoDataSource_CreatorDetailsLoaded_ExperimentalVariant() {
     let project = Project.template
 
     let envelope = ProjectCreatorDetailsEnvelope(backingsCount: 25, id: "123", launchedProjectsCount: 50)
 
     let mockService = MockService(fetchProjectCreatorDetailsResult: .success(envelope))
 
-    withEnvironment(apiService: mockService) {
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~ [
+        OptimizelyExperiment.Key.nativeProjectPageConversionCreatorDetails.rawValue: OptimizelyExperiment.Variant.variant1.rawValue
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/8676c26a01ecffa0f982d0e5b09f2191cd795621/Library/ViewModels/ProjectPamphletContentViewModelTests.swift#L410"></span>Line should be 110 characters or less: currently 131 characters<br />
<code>line_length</code> <code>ProjectPamphletContentViewModelTests.swift:410</code></td>
    </tr>
  </tbody>
</table>